src/karmen_frontend/src/components/printers/webcam-stream.js

```diff
@@ -39,6 +40,49 @@ const WebcamModal = ({ classNames, source, url, allowFullscreen }) => {
   );
 };
 
+export const WebcamStreamRenderer = (props) => {
+  const {
```

Comment:
Please expand props directly in the func definition. This way, IDEs will be able to give devs suggestions. E.g.:

```javascript
export const WebcamStreamRenderer = ({
    url,
    image,
    flipHorizontal,
    flipVertical,
    rotate90,
    allowFullscreen,
    printer
}) => {
...
}
```

Reply:
Done.